spec/mutations/guides/create_guide_spec.rb

```diff
@@ -6,28 +6,38 @@
   let(:params) do
     { user:    FactoryGirl.create(:user),
       crop_id: FactoryGirl.create(:crop).id.to_s,
-      name:    'hi.',
-      id:      FactoryGirl.create(:guide).id.to_s }
+      id:      FactoryGirl.create(:guide).id.to_s,
+      attributes: {
+        name:    'hi.',
+        } }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.